bin/uglifyjs

```diff
@@ -204,6 +205,15 @@ function run() {
     } catch (ex) {
         fatal(ex);
     }
+    if (program.enclose != null) {
```

Comment:
This is the wrong place for processing `options` (hint: see where `define` is processed).